README.md

```diff
@@ -1,6 +1,49 @@
-# apphub
+# AppHub 
 
-## Project setup
+[![Known Vulnerabilities](https://snyk.io/test/github/esdc-devx/AppHub/badge.svg?targetFile=app-hub-api%2Fpackage.json)](https://snyk.io/test/github/esdc-devx/AppHub?targetFile=app-hub-api%2Fpackage.json)
+[![Build Status](https://dev.azure.com/dx-team/AppHub/_apis/build/status/esdc-devx.AppHub?branchName=master)](https://dev.azure.com/dx-team/AppHub/_build/latest?definitionId=2&branchName=master)
+
+A dashboard for displaying information on systems department wide. 
+
+## Run Integration and a11y Tests
+
+Run the following command in the `integration-tests` folder. 
+
+### Setup Tests
+```
+npm ci
+```
+
+### Run Tests
+Run the following command to test the app in electron headless mode
+```
+npx cypress run
+```
```

Comment:
Once I was actually able to get this running do to missing dependencies https://docs.cypress.io/guides/guides/continuous-integration.html#Advanced-setup
 It's still gives me errors because it can't connect to http://localhost:4848 - I'm guessing the app needs to be run in docker first to run the test, so that should be mentioned here.

Reply:
I updated the readme

Reply:
Did you push your change, i don't see it...